src/components/VolunteerDetailContracts.vue

```diff
@@ -0,0 +1,205 @@
+<template>
+  <div class="mt-0">
+    <header class="flex justify-end mt-8">
+      <!-- <SearchBar placeholder="Suche nach Vereinbarungen" /> -->
+      <ButtonStandard>Vereinbarung hinzufügen</ButtonStandard>
+    </header>
+
+    <!-- Grid -->
+    <div v-if="volunteer && contract" class="flex flex-wrap gap-[10px] mt-8">
+      <!-- Contract-Overview -->
+      <section class="rounded bg-white px-[18px] py-[16px] w-[330px] h-[120px]">
+        <h2 class="inline font-medium">Vereinbarung</h2>
+        <p class="inline text-vologray-400">
+          - vom
+          {{ new Date(contract.created).toLocaleDateString() }}
+        </p>
+        <!-- Pills -->
+        <div class="mt-2 flex gap-2 flex-wrap items-center">
+          <div
+            class="inline bg-voloblue-200 px-2 pt-[1px] rounded-2xl flex justify-center items-center"
+          >
+            <p class="inline block text-white">
+              {{ contract.program }}
+            </p>
+          </div>
+          <div
+            v-if="contract.visaNecessary"
+            class="inline bg-voloyellow-100 px-2 pt-[1px] rounded-2xl flex justify-center items-center"
+          >
+            <p class="inline block">Visum</p>
+          </div>
+          <div
+            v-if="contract.incomingVolunteer"
+            class="inline bg-voloyellow-100 px-2 pt-[1px] rounded-2xl flex justify-center items-center"
+          >
+            <p class="inline block">Incoming</p>
+          </div>
+          <div
+            v-if="age < 18"
+            class="inline bg-voloyellow-100 px-2 pt-[1px] rounded-2xl flex justify-center items-center"
+          >
+            <p class="inline block">U18</p>
+          </div>
+          <div
+            v-if="age > 25"
+            class="inline bg-voloyellow-100 px-2 pt-[1px] rounded-2xl flex justify-center items-center"
+          >
+            <p class="inline block">Ü25</p>
+          </div>
+          <button
+            class="inline bg-vologray-300 text-white px-2 rounded-2xl flex justify-center items-center"
+          >
+            <p class="inline block">+</p>
```

Comment:
`<p class="inline block">+</p>` in my opinion this is not pretty and visibly uneven and i would choose an icon like this: 
https://icones.js.org/collection/ph?s=add&icon=ph:plus-circle-fill

Reply:
✅